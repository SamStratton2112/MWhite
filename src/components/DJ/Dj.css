.dj {
    /* background-image: url(https://i.imgur.com/rAQGuxo.jpeg); */
    background-position: center;
    background-repeat: no-repeat;
    background-attachment: fixed;
    width: 100%;
    /* background-color:rgba(8, 8, 8, 0.9); */
    background-size: cover;
}