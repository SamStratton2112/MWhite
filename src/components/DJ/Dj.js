import { Container } from "react-bootstrap";
import MusicContainer from "./MusicContainer";
import PrevExperience from "./PrevExperience";
import AppearencesContainer from "./AppearencesContainer";
import DjCarousel from "./DjCarousel";
import Footer from '../Footer'

const Dj = () => {
    return (
        <>
            <Container className="cardText" style={{ backgroundColor: 'rgb(13, 12, 12, .85)'}}>
                <br />
                <Container style={{width: '10em', borderRadius: '.2em' }}>
                    <h3 style={{ padding: '.5em' }} >Albums</h3>
                </Container>
                <MusicContainer />
                <Container style={{width: '20em', borderRadius: '.2em' }}>
                    <h3 style={{ padding: '.5em', marginTop: '.5em' }}>Venues Played</h3>
                </Container>
                <PrevExperience />
                <Container style={{width: '15em', borderRadius: '.2em' }}>
                    <h3 style={{ padding: '.5em', marginBottom: '.5em', marginTop: '1.5em' }}>Appearances...</h3>
                </Container>
                <AppearencesContainer />
                <DjCarousel />
            </Container>
            <Footer />
        </>
    )
};

export default Dj;