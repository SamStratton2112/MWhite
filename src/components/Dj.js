import { Container } from "react-bootstrap";
import MusicContainer from "./MusicContainer";
import PrevExperience from "./PrevExperience";
import AppearencesContainer from "./AppearencesContainer";

const Dj = ()=>{
    return(
        <Container>
            <span>Check out my albums:</span>
            <MusicContainer/>
            <PrevExperience/>
            <AppearencesContainer/>
        </Container>
    )
};

export default Dj;