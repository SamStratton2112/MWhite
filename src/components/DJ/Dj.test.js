import Dj from './Dj';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

describe(Dj, () => {
    it("renders without crashing", () => {
        render(<MemoryRouter>
            <Dj />
        </MemoryRouter>);
    });
    it("matches snapshot", () => {
        const { asFragment } = render(<MemoryRouter>
            <Dj />
        </MemoryRouter>);
        expect(asFragment()).toMatchSnapshot();
    });
});