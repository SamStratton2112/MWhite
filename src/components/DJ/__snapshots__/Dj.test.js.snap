// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dj matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="dj cardText container"
  >
    <br />
    <div
      class="container"
      style="width: 20em; border-radius: .2em;"
    >
      <h3
        style="padding: 1em; margin-bottom: 1em;"
      >
        Albums
      </h3>
    </div>
    <div
      class="center  container"
    >
      <div
        class="row row-cols-md-12"
      >
        <div
          class="col"
        >
          <div
            class="center card"
            style="color: white; margin: 1em; border: solid; border-color: rgb(184, 17, 17);"
          >
            <img
              alt="Unwind"
              class="card-img-top"
              src="https://i.imgur.com/WAey4uy.jpeg"
              style="width: 15em; height: 16em; padding-top: 1em;"
            />
            <div
              class="card-body"
            >
              <div
                class="card-title h5"
                style="font-size: 1.7em;"
              >
                Unwind
              </div>
              <a
                href="https://soundcloud.com/user-554826787/unwind"
              >
                <button
                  class="bttn"
                >
                  Listen on Soundcloud
                </button>
              </a>
            </div>
          </div>
        </div>
        <div
          class="col"
        >
          <div
            class="center card"
            style="color: white; margin: 1em; border: solid; border-color: rgb(184, 17, 17);"
          >
            <img
              alt="Sexy Time"
              class="card-img-top"
              src="https://i.imgur.com/VMQqEA7.jpeg"
              style="width: 15em; height: 16em; padding-top: 1em;"
            />
            <div
              class="card-body"
            >
              <div
                class="card-title h5"
                style="font-size: 1.7em;"
              >
                Sexy Time
              </div>
              <a
                href="https://soundcloud.com/user-554826787/sexytime"
              >
                <button
                  class="bttn"
                >
                  Listen on Soundcloud
                </button>
              </a>
            </div>
          </div>
        </div>
        <div
          class="col"
        >
          <div
            class="center card"
            style="color: white; margin: 1em; border: solid; border-color: rgb(184, 17, 17);"
          >
            <img
              alt="An inspired album..."
              class="card-img-top"
              src="https://i.imgur.com/fHMsQky.jpeg"
              style="width: 15em; height: 16em; padding-top: 1em;"
            />
            <div
              class="card-body"
            >
              <div
                class="card-title h5"
                style="font-size: 1.7em;"
              >
                An inspired album...
              </div>
              <a
                href="https://soundcloud.com/user-554826787/an-album-inspired-by-kruder-dorfmeister-mixed-by-dj-mike-white"
              >
                <button
                  class="bttn"
                >
                  Listen on Soundcloud
                </button>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="container"
      style="width: 20em; border-radius: .2em;"
    >
      <h3
        style="padding: 1em;"
      >
        Venues Played
      </h3>
    </div>
    <div
      class="center container"
    >
      <div
        class="row row-cols-lg-3 row-cols-md-12"
      >
        <div
          class="col"
        >
          <div
            class="col"
          >
            <p
              style="border-radius: .2em; margin-top: 1em; width: 12em;"
            >
              San Diego, California
            </p>
            <div
              class="container"
              style="width: 20rem; height: 20rem;"
            >
              <img
                alt="San Diego, California"
                src="https://i.imgur.com/2O7KGP6.jpeg"
                style="width: 20rem; height: 20rem; border-radius: .5em;"
              />
            </div>
          </div>
        </div>
        <div
          class="col"
        >
          <div
            class="col"
          >
            <p
              style="border-radius: .2em; margin-top: 1em; width: 12em;"
            >
              Las Vegas, Nevada
            </p>
            <div
              class="container"
              style="width: 20rem; height: 20rem;"
            >
              <img
                alt="Las Vegas, Nevada"
                src="https://i.imgur.com/WYPhHOV.jpeg"
                style="width: 20rem; height: 20rem; border-radius: .5em;"
              />
            </div>
          </div>
        </div>
        <div
          class="col"
        >
          <div
            class="col"
          >
            <p
              style="border-radius: .2em; margin-top: 1em; width: 12em;"
            >
              Los Angeles, California
            </p>
            <div
              class="container"
              style="width: 20rem; height: 20rem;"
            >
              <img
                alt="Los Angeles, California"
                src="https://i.imgur.com/uLfW5z0.png"
                style="width: 20rem; height: 20rem; border-radius: .5em;"
              />
            </div>
          </div>
        </div>
        <div
          class="col"
        >
          <div
            class="col"
          >
            <p
              style="border-radius: .2em; margin-top: 1em; width: 12em;"
            >
              Palm Springs, California
            </p>
            <div
              class="container"
              style="width: 20rem; height: 20rem;"
            >
              <img
                alt="Palm Springs, California"
                src="https://i.imgur.com/soZ7oyi.jpeg"
                style="width: 20rem; height: 20rem; border-radius: .5em;"
              />
            </div>
          </div>
        </div>
        <div
          class="col"
        >
          <div
            class="col"
          >
            <p
              style="border-radius: .2em; margin-top: 1em; width: 12em;"
            >
              New York, New York
            </p>
            <div
              class="container"
              style="width: 20rem; height: 20rem;"
            >
              <img
                alt="New York, New York"
                src="https://i.imgur.com/3JBkh6t.jpeg"
                style="width: 20rem; height: 20rem; border-radius: .5em;"
              />
            </div>
          </div>
        </div>
        <div
          class="col"
        >
          <div
            class="col"
          >
            <p
              style="border-radius: .2em; margin-top: 1em; width: 12em;"
            >
              Charlotte, North Carolina
            </p>
            <div
              class="container"
              style="width: 20rem; height: 20rem;"
            >
              <img
                alt="Charlotte, North Carolina"
                src="https://i.imgur.com/Oqdr1Ty.jpeg"
                style="width: 20rem; height: 20rem; border-radius: .5em;"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="container"
      style="width: 20em; border-radius: .2em;"
    >
      <h3
        style="padding: 1em; margin-top: 1em; margin-bottom: 1em;"
      >
        Appearences...
      </h3>
    </div>
    <div
      class="container"
    >
      <button
        class="bttn"
      >
        As a Guest D.J.
      </button>
      <button
        class="bttn"
      >
        As an Opening D.J.
      </button>
      <button
        class="bttn"
      >
        As a Featured D.J.
      </button>
    </div>
    <div
      class="container"
      style="margin-top: 4em; padding-bottom: 10em;"
    >
      <div
        class="align-items-center carousel slide"
      >
        <div
          class="carousel-indicators"
        >
          <button
            aria-current="true"
            aria-label="Slide 1"
            class="active"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 2"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 3"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 4"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 5"
            data-bs-target=""
            type="button"
          />
        </div>
        <div
          class="carousel-inner"
        >
          <div
            class="active carousel-item"
            interval="5000"
          >
            <div
              class="container"
              style="width: 50%;"
            >
              <div
                class="ratio ratio-16x9"
              >
                <iframe
                  allowfullscreen=""
                  src="https://www.youtube.com/embed/BKKLGFTLobs?rel=0"
                  title="YouTube video"
                />
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
            interval="5000"
          >
            <div
              class="container"
              style="width: 50%;"
            >
              <div
                class="ratio ratio-16x9"
              >
                <iframe
                  allowfullscreen=""
                  src="https://www.youtube.com/embed/drABef-QGhg?rel=0"
                  title="YouTube video"
                />
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
            interval="5000"
          >
            <div
              class="container"
              style="width: 50%;"
            >
              <div
                class="ratio ratio-16x9"
              >
                <iframe
                  allowfullscreen=""
                  src="https://www.youtube.com/embed/vz0OvRqfbro?rel=0"
                  title="YouTube video"
                />
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
            interval="5000"
          >
            <div
              class="container"
              style="width: 50%;"
            >
              <div
                class="ratio ratio-16x9"
              >
                <iframe
                  allowfullscreen=""
                  src="https://www.youtube.com/embed/MNHSMyaNZ6Y?rel=0"
                  title="YouTube video"
                />
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
            interval="5000"
          >
            <div
              class="container"
              style="width: 50%;"
            >
              <div
                class="ratio ratio-16x9"
              >
                <iframe
                  allowfullscreen=""
                  src="https://www.youtube.com/embed/5ZcIKeqHBuo?rel=0"
                  title="YouTube video"
                />
              </div>
            </div>
          </div>
        </div>
        <a
          class="carousel-control-prev"
          href="#"
          role="button"
          tabindex="0"
        >
          <span
            aria-hidden="true"
            class="carousel-control-prev-icon"
          />
          <span
            class="visually-hidden"
          >
            Previous
          </span>
        </a>
        <a
          class="carousel-control-next"
          href="#"
          role="button"
          tabindex="0"
        >
          <span
            aria-hidden="true"
            class="carousel-control-next-icon"
          />
          <span
            class="visually-hidden"
          >
            Next
          </span>
        </a>
      </div>
    </div>
  </div>
  <div
    class="containerBckgroundRed container"
  >
    <p
      style="font-size: larger;"
    >
      _____________________
    </p>
    <img
      alt=" DJ Mike White Logo"
      src="https://i.imgur.com/QLkSBI3.jpeg"
      style="width: 20em;"
    />
    <div
      class="center container"
    >
      <a
        class="socialMedia"
        href="https://soundcloud.com/user-554826787"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-soundcloud fa-2x "
          data-icon="soundcloud"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 640 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M639.8 298.6c-1.3 23.1-11.5 44.8-28.4 60.5s-39.2 24.4-62.3 24.1h-218c-4.8 0-9.4-2-12.8-5.4s-5.3-8-5.3-12.8V130.2c-.2-4 .9-8 3.1-11.4s5.3-6.1 9-7.7c0 0 20.1-13.9 62.3-13.9c25.8 0 51.1 6.9 73.3 20.1c17.3 10.2 32.3 23.8 44.1 40.1s20 34.8 24.2 54.4c7.5-2.1 15.3-3.2 23.1-3.2c11.7-.1 23.3 2.2 34.2 6.7S606.8 226.6 615 235s14.6 18.3 18.9 29.3s6.3 22.6 5.9 34.3zm-354-153.5c.1-1 0-2-.3-2.9s-.8-1.8-1.5-2.6s-1.5-1.3-2.4-1.7s-1.9-.6-2.9-.6s-2 .2-2.9 .6s-1.7 1-2.4 1.7s-1.2 1.6-1.5 2.6s-.4 1.9-.3 2.9c-6 78.9-10.6 152.9 0 231.6c.2 1.7 1 3.3 2.3 4.5s3 1.8 4.7 1.8s3.4-.6 4.7-1.8s2.1-2.8 2.3-4.5c11.3-79.4 6.6-152 0-231.6zm-44 27.3c-.2-1.8-1.1-3.5-2.4-4.7s-3.1-1.9-5-1.9s-3.6 .7-5 1.9s-2.2 2.9-2.4 4.7c-7.9 67.9-7.9 136.5 0 204.4c.3 1.8 1.2 3.4 2.5 4.5s3.1 1.8 4.8 1.8s3.5-.6 4.8-1.8s2.2-2.8 2.5-4.5c8.8-67.8 8.8-136.5 .1-204.4zm-44.3-6.9c-.2-1.8-1-3.4-2.3-4.6s-3-1.8-4.8-1.8s-3.5 .7-4.8 1.8s-2.1 2.8-2.3 4.6c-6.7 72-10.2 139.3 0 211.1c0 1.9 .7 3.7 2.1 5s3.1 2.1 5 2.1s3.7-.7 5-2.1s2.1-3.1 2.1-5c10.5-72.8 7.3-138.2 .1-211.1zm-44 20.6c0-1.9-.8-3.8-2.1-5.2s-3.2-2.1-5.2-2.1s-3.8 .8-5.2 2.1s-2.1 3.2-2.1 5.2c-8.1 63.3-8.1 127.5 0 190.8c.2 1.8 1 3.4 2.4 4.6s3.1 1.9 4.8 1.9s3.5-.7 4.8-1.9s2.2-2.8 2.4-4.6c8.8-63.3 8.9-127.5 .3-190.8zM109 233.7c0-1.9-.8-3.8-2.1-5.1s-3.2-2.1-5.1-2.1s-3.8 .8-5.1 2.1s-2.1 3.2-2.1 5.1c-10.5 49.2-5.5 93.9 .4 143.6c.3 1.6 1.1 3.1 2.3 4.2s2.8 1.7 4.5 1.7s3.2-.6 4.5-1.7s2.1-2.5 2.3-4.2c6.6-50.4 11.6-94.1 .4-143.6zm-44.1-7.5c-.2-1.8-1.1-3.5-2.4-4.8s-3.2-1.9-5-1.9s-3.6 .7-5 1.9s-2.2 2.9-2.4 4.8c-9.3 50.2-6.2 94.4 .3 144.5c.7 7.6 13.6 7.5 14.4 0c7.2-50.9 10.5-93.8 .3-144.5zM20.3 250.8c-.2-1.8-1.1-3.5-2.4-4.8s-3.2-1.9-5-1.9s-3.6 .7-5 1.9s-2.3 2.9-2.4 4.8c-8.5 33.7-5.9 61.6 .6 95.4c.2 1.7 1 3.3 2.3 4.4s2.9 1.8 4.7 1.8s3.4-.6 4.7-1.8s2.1-2.7 2.3-4.4c7.5-34.5 11.2-61.8 .4-95.4z"
            fill="currentColor"
          />
        </svg>
      </a>
      <a
        class="socialMedia"
        href="https://www.instagram.com/djmikewhite_/"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-instagram fa-2x "
          data-icon="instagram"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
            fill="currentColor"
          />
        </svg>
      </a>
      <a
        class="socialMedia"
        href="https://www.facebook.com/mike.white.7739814"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-square-facebook fa-2x "
          data-icon="square-facebook"
          data-prefix="fab"
          focusable="false"
          role="img"
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M64 32C28.7 32 0 60.7 0 96V416c0 35.3 28.7 64 64 64h98.2V334.2H109.4V256h52.8V222.3c0-87.1 39.4-127.5 125-127.5c16.2 0 44.2 3.2 55.7 6.4V172c-6-.6-16.5-1-29.6-1c-42 0-58.2 15.9-58.2 57.2V256h83.6l-14.4 78.2H255V480H384c35.3 0 64-28.7 64-64V96c0-35.3-28.7-64-64-64H64z"
            fill="currentColor"
          />
        </svg>
      </a>
    </div>
    <p
      style="text-align: right;"
    >
      <a
        href="https://sam-stratton2112.onrender.com"
        style="text-decoration-color: white; color: white; margin-bottom: 2em;"
      >
        © 2024 Samantha Stratton
      </a>
    </p>
  </div>
</DocumentFragment>
`;
